app/views/devise/sessions/new.html.erb

```diff
@@ -38,6 +38,7 @@
             </div>
             <div class="" id="login_button">
               <%= f.button :submit, "Log in", class: "button is-primary" %>
+              <%= link_to "Sign Up", new_user_registration_path, class: "button is-success" %>
```

Comment:
I think this should probably not live inside the `#login_button` div and instead be in a separate sibling div.

Reply:
Thank you for reminding me. I'll fix this one and merge.